src/test/java/it/eng/idsa/businesslogic/service/impl/DapsV2ServiceImplTest.java

```diff
@@ -133,6 +134,39 @@ public void validateTokenFailed() throws ParseException {
 		assertFalse(valid);
 	}
 	
+	@Test
+	public void validateEmptyTokenFailed() throws ParseException {
+		String tokenValue = "";
+		
+		when(dapsUtilityProvider.provideAlgorithm(tokenValue)).thenReturn(algorithm);
+		doThrow(JWTDecodeException.class).when(algorithm).verify(any(DecodedJWT.class));
+		
+		boolean valid = dapsV2Service.validateToken(tokenValue);
+		assertFalse(valid);
+	}
+	
+	@Test
+	public void validateBlankTokenFailed() throws ParseException {
+		String tokenValue = " ";
+		
+		when(dapsUtilityProvider.provideAlgorithm(tokenValue)).thenReturn(algorithm);
+		doThrow(JWTDecodeException.class).when(algorithm).verify(any(DecodedJWT.class));
+		
+		boolean valid = dapsV2Service.validateToken(tokenValue);
+		assertFalse(valid);
+	}
+
+	@Test
+	public void validateWrongJsonFormatTokenFailed() throws ParseException {
+		String tokenValue = JwTokenUtil.generateToken(false);
```

Comment:
Was intention to test here
String tokenValue = "ABC";
If yes, please also rename test, and remove Json from method name (validateWrongTokenFormatFailed).